examples/web-wagmi/src/profiles/UseUpdateProfileDetails.tsx

```diff
@@ -0,0 +1,111 @@
+import {
+  ProfileFieldsFragment,
+  useActiveProfile,
+  useUpdateProfileDetails,
+} from '@lens-protocol/react';
+import { useForm } from 'react-hook-form';
+
+import { upload } from '../upload';
+import { ProfileCard } from './components/ProfileCard';
+
+type UpdateProfileFormProps = {
+  profile: ProfileFieldsFragment;
+};
+
+type FormData = {
+  name: string;
+  bio: string | null;
+  attributes: {
+    location: string | null;
+    website: string | null;
+  };
+};
+
+function UpdateProfileForm({ profile }: UpdateProfileFormProps) {
+  const { register, handleSubmit } = useForm<FormData>();
+  const { update, error, isPending } = useUpdateProfileDetails({ profile, upload });
+
+  const onSubmit = handleSubmit(async ({ name, bio, attributes }) => {
+    await update({ name, bio, attributes });
+  });
+
+  return (
+    <form onSubmit={onSubmit}>
+      <ProfileCard profile={profile} />
+
+      <label>
+        Name:
+        <br />
+        <input
+          required
+          type="text"
+          placeholder="Your name"
+          disabled={isPending}
+          defaultValue={profile.name ?? ''}
+          {...register('name')}
+        />
+      </label>
+
+      <label>
+        Bio:
+        <br />
+        <textarea
+          rows={3}
+          placeholder="Write a line about you"
+          style={{ resize: 'none' }}
+          disabled={isPending}
+          defaultValue={profile.bio ?? ''}
+          {...register('bio')}
+        ></textarea>
+      </label>
+
+      <label>
+        Location:
+        <br />
+        <input
+          required
+          type="text"
+          placeholder="Where are you?"
+          disabled={isPending}
+          defaultValue={profile.attributes.location.toString()}
```

Comment:
I'm seeing an error thrown on this line when running locally with my profile `reecej.test`

Seems I don't have any attributes set

![image](https://user-images.githubusercontent.com/34844044/212883528-e21bfc12-a635-4607-9edc-04d7e1072a48.png)

Reply:
could it be a not-recompiled `api-bindings`?